clang/lib/StaticAnalyzer/Checkers/WebKit/UncountedCallArgsChecker.cpp

```diff
@@ -54,7 +54,7 @@ class UncountedCallArgsChecker
       bool shouldVisitImplicitCode() const { return false; }
 
       bool TraverseDecl(Decl *D) {
-        if (isa<ClassTemplateDecl>(D) && isRefType(safeGetName(D)))
+        if (D && isa<ClassTemplateDecl>(D) && isRefType(safeGetName(D)))
           return true;
         return RecursiveASTVisitor<LocalVisitor>::TraverseDecl(D);
       }
```

Comment:
Shouldn't this be overriding `TraverseTemplateInstantiations(ClassTemplateDecl *D)` instead?
    
 `RecursiveASTVisitor<LocalVisitor>::TraverseDecl`  will ensure you won't be visiting null nodes.

Reply:
Good point. That would be cleaner.